packages/outputarea/test/widget.spec.ts

```diff
@@ -376,6 +376,14 @@ describe('outputarea/widget', () => {
           const factory = new OutputArea.ContentFactory();
           const future = kernel.requestExecute({ code: CODE });
           const options = {
+            parent_header: {
```

Comment:
This doesn't have any effect, right? The options passed to createStdin are very well specified, and do not include the parent header (and the parent header should be the header from the request message anyway).

It seems like the kernel connection sendInputReply method needs an option for the parent header, and then that needs to be threaded through calls to make its way down to it.

Reply:
> The options passed to createStdin are very well specified, and do not include the parent header

I added `parent_header` to the options passed to `createStdin` [here](https://github.com/jupyterlab/jupyterlab/pull/12376/files#diff-4d0f96dfb060e341e538a737407cb92ad9a1003d88037f844a289e11134837abR954), but I'm not sure it's the way to go.